pkg/k8sapi/k8sutils.go

```diff
@@ -37,7 +37,11 @@ func getIPAMDCacheFilters() map[client.Object]cache.ByObject {
 		return map[client.Object]cache.ByObject{
 			&corev1.Pod{}: {
 				Field: fields.Set{"spec.nodeName": nodeName}.AsSelector(),
-			}}
+			},
```

Comment:
The previous two related to changes to the cache improvements were these.

- https://github.com/aws/amazon-vpc-cni-k8s/pull/1419/files#diff-f48443f100e2f9072cbb1924ed0a9090831ade5986d10535641ad5c0a63f51b4
- https://github.com/aws/amazon-vpc-cni-k8s/commit/ea10123e8105df97d2b2fd9916fc7d255458075c#diff-f48443f100e2f9072cbb1924ed0a9090831ade5986d10535641ad5c0a63f51b4

I am trying to understand why the only `Pod{}` to the filter here in the first place.

Reply:
I suspect it wasn't as obvious because this requires large clusters to show the increase in memory and networking bandwidth. We only saw this because our cluster is 4k node count